server/controllers/request.js

```diff
@@ -26,8 +27,9 @@ exports.getRequests = asyncHandler(async (req, res, next) => {
 // @access Private
 exports.createRequest = asyncHandler(async (req, res, next) => {
   const { id: ownerId } = req.user;
-  let { sitterId, start, end } = req.body;
-
+  let { profileId, start, end } = req.body;
+  const userByProfile = await Profile.findById(profileId, "userId");
+  const sitterId = userByProfile.userId;
```

Comment:
See comments above. If those changes are implemented this will not be needed. simply revert to the original implementation 🙂